pom.xml

```diff
@@ -82,7 +82,7 @@
         <plugins.nexus-staging.version>1.6.8</plugins.nexus-staging.version>
         <plugins.release.version>2.5.3</plugins.release.version>
         <plugins.source.version>3.0.1</plugins.source.version>
-        <plugins.surefire.version>3.0.0-M3</plugins.surefire.version>
+        <plugins.surefire.version>2.22.2</plugins.surefire.version>
```

Comment:
Was this dependency manually downgraded? If so, I'm curious as to why.

Reply:
On another project, I tried to upgrade to a milestone release an received pushback. My thought was that we never should have upgraded unless we had a reason, so I thought the previous upgrade was a mistake. Happy to return to a milestone release if that's what is wanted.

Reply:
No, I was just curious. As long as the tests all pass, it should be fine. It looks like there aren't any status checks on this project, have the tests been run?

Reply:
The tests passed both locally and on Bamboo.